README.md

```diff
@@ -49,6 +49,59 @@ This project uses the [pyrex](https://github.com/garmin/pyrex) tool to transpare
 
     <font color=lightgreen>[NI]</font> builders who are connected to the NI  corporate network should specify `-org` in their init script args, to provoke the script into adding the `ni-org.conf` snippet to your bitbake directory. External builders *should not* use `--org`.
 
+5. Build the package or packages that you want for your target. For example, to build Python, Ruby, and Apache for Zynq targets, run the following commands:
+
+        bitbake python ruby apache2
+
+    To build every package in NI's feed, run the following commands:
+
+        bitbake packagefeed-ni-core
+        bitbake --continue packagefeed-ni-extra
+
+    **NOTE** The configuration files (build/conf/*.conf) can optionally be changed to reflect the desired build settings instead of setting environment variables.
+
+    **NOTE** Building packages through OpenEmbedded can use significantdisk space, on the order of tens of gigabytes. If you are preparing a virtual machine to build images, make sure to allocate sufficient disk space.
+
+    The resulting ipk files that can be installed through opkg exist at
+the following directory:
+
+        tmp-glibc/deploy/ipk/...
+        
+6. Bitbake can transform tmp-glibc/deploy/ipk/ into a package feed when you run the following command:
+
+        bitbake package-index
+
+    You must rebuild the package-index before building images or if any package whom the images you are trying to build depends on has been changed and rebuilt.
+    
+7. Building various images
+
+    Build a safemode image by running the following commands:
+
+        bitbake nilrt-safemode-rootfs
+    
+    Build a runmode image by running the following commands:
+
+        bitbake nilrt-runmode-rootfs
+
+    Build a bootable recovery media by running the following commands:
+
+        bitbake nilrt-recovery-media
+
+    **NOTE** You must build everything in packagefeed-ni-core(step 5) and build package_index (step 6) to build images.
```

Comment:
This isn't strictly true as `packagefeed-ni-extra` is not necessary for building images - only `packagefeed-ni-core` is required.

Reply:
Yeah, I meant to say packagefeed-ni-core is the only one needed. Let me reword this a bit.

Reply:
Fixed.